docs/pages/components/loading-spinner.md

```diff
@@ -41,7 +41,7 @@ The spinner should be included inside the container. Visibility can be toggled i
         <div></div>
     </div>
     <div class="fd-panel__header">
-        <h1 class="fd-panel__title">Lorem ipsum</h1>
+        <h3 class="fd-panel__title">Lorem ipsum</h3>
```

Comment:
Should this page have the block quote disclaimer (`site.data.strings.headerDisclaimer`)?

Reply:
Same reasoning as contextual menu, this is actually using a panel component.

Reply:
Good point.